### DISCUSSION TITLE
Meta: Labels for runtime-specific issues (e.g. podman)?

### DISCUSSION THREAD

**Discussion by LewisGaul** [gabrieldemarmiesse/python-on-whales#515] [General]
The docs state POW can be used with podman and nerdctl (https://gabrieldemarmiesse.github.io/python-on-whales/#alternatives-to-docker-podman-nerdctl), and this is true, I have [tests](https://github.com/ios-xr/xrd-ha-app/tree/main/tests/) that make use of this (exposed via `--container-exe` arg to the tests). In fact, these days I almost entirely use podman rather than docker, and my expectation would be that it continues to gain popularity (the default in RHEL, and perhaps Fedora? relevant blog post: https://fedoramagazine.org/docker-and-fedora-37-migrating-to-podman/).

However, there are occasionally POW issues specific to using podman (a few that I've fixed, e.g. from [this search](https://github.com/gabrieldemarmiesse/python-on-whales/issues?q=author%3ALewisGaul+%22podman%22+is%3Amerged)).

It might be helpful to be able to track/classify these issues with a label?

I have a few things to suggest (see https://github.com/gabrieldemarmiesse/python-on-whales/discussions/516), but not wanting to pollute the "global namespace" of issues (wanting to make it clear they don't affect using POW with docker).

**Comment by gabrieldemarmiesse** [comment]
Sure, feel free to make a new label for podman :)
